lib/services/http_service.dart

```diff
@@ -0,0 +1,132 @@
+
```

Comment:
this class needs to be broken down and each model extracted to its own class

Reply:
You mean each model class needs to be extracted to separate file?
I prefer keep them in same file with service as long they are not used by other services.
In my understanding, those models are just like factory functions constructing objects with specific fields and are not intended to implement custom logic.

Reply:
It's a matter of convention as all the open-source app in flutter and other languages try to separate data classes into their own package see
https://github.com/appditto/natrium_wallet_flutter
https://github.com/roughike/inKino